Source/Android/app/src/main/res/values/arrays.xml

```diff
@@ -33,6 +33,26 @@
         <item>0</item>
     </integer-array>
 
+    <!-- DSP Emulation Engine -->
+    <string-array name="dspEngineEntriesX86_64" translatable="false">
+        <item>DSP HLE Emulation (fast)</item>
```

Comment:
Nitpick, but this string suffers from RAS syndrome. I suggest removing Emulation:

```suggestion
        <item>DSP HLE (fast)</item>
```

Reply:
This is what we use in DolphinQt, though... I am planning to add translations for Android at some point, and when we get that, it's good if the strings are identical between Android and DolphinQt so that the translators don't get any extra work.

Reply:
Oh, DolphinQt has the same problem :thinking: 

Do you think we should update the DolphinQt string too, then? Or leave it as is?

Reply:
It would make sense to open a new PR with that change, I think.